### ISSUE TITLE
SQL exec xp_cmdshell

### ISSUE THREAD

**Issue by Jlarm90** [microsoft/azuredatastudio#23194]
Issue Type: <b>Bug</b>

EXEC master.dbo.xp_cmdshell @CMD
does not work on Azure Data Studio but does on SSMS.
suspicion is that the command is ran with the current logged in crendentials and not the service account.

Azure Data Studio version: azuredatastudio 1.37.0 (d904740d93d7df76a0ba361f20e4351813b57645, 2022-06-14T01:05:12.352Z)
OS version: Windows_NT x64 10.0.19045
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 PRO 4650U with Radeon Graphics (12 x 2096)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|7.33GB (0.74GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details><details><summary>Extensions (6)</summary>

Extension|Author (truncated)|Version
---|---|---
ssmskeymap|kev|1.1.0
admin-pack|Mic|0.0.2
agent|Mic|0.49.0
dacpac|Mic|1.13.0
import|Mic|1.5.4
profiler|Mic|0.12.2

(1 theme extensions excluded)

</details>

**Comment by kburtram** [comment]
Thanks for reporting this issue!  I was able to execute `EXEC master.dbo.xp_cmdshell` within Azure Data Studio when I logged in with a SqlLogin account with the required permissions.  Is the bug specific to `EXEC master.dbo.xp_cmdshell` or that the connection isn't picking up the credentials you expected (when using Integrated Authentication)?

![image](https://github.com/microsoft/azuredatastudio/assets/599935/6e4020f3-0fff-448d-aea3-1e68ff375b4a)
